types/stripe-js/stripe.d.ts

```diff
@@ -941,6 +985,10 @@ export type SetupIntentResult =
   | {setupIntent: api.SetupIntent; error?: undefined}
   | {setupIntent?: undefined; error: StripeError};
 
+export type OrderResult =
+  | {paymentIntent: api.PaymentIntent; order: api.Order; error?: undefined}
```

Comment:
For $0 Orders submitted through processOrder, they won't have a payment intent. Should we add another type here thats `{paymentIntent?: undefined; order: api.Order; error?: undefined}`

Reply:
good callout! will add this type